test/message-stream.ts

```diff
@@ -142,7 +144,7 @@ describe('MessageStream', () => {
     const gaxClient = new FakeGaxClient();
     client = gaxClient.client;  // we hit the grpc client directly
     subscriber = new FakeSubscriber(gaxClient);
-    messageStream = new MessageStream(subscriber);
+    messageStream = new MessageStream(subscriber as {} as Subscriber);
```

Comment:
I'm seeing this cast all over the test file. Why not just cast the subscriber in the line above?

Reply:
Done